x-pack/plugins/apm/server/lib/service_map/get_service_map.ts

```diff
@@ -129,58 +132,31 @@ async function getServicesData(options: IEnvOptions) {
   );
 }
 
-function getAnomaliesData(options: IEnvOptions) {
-  const { start, end, client } = options.setup;
-  const rangeQuery = { range: rangeFilter(start, end, 'timestamp') };
-
-  const params = {
-    index: getMlIndex('*'),
-    body: {
-      size: 0,
-      query: {
-        bool: { filter: [{ term: { result_type: 'record' } }, rangeQuery] },
-      },
-      aggs: {
-        jobs: {
-          terms: { field: 'job_id', size: 10 },
-          aggs: {
-            top_score_hits: {
-              top_hits: {
-                sort: [{ record_score: { order: 'desc' as const } }],
-                _source: ['job_id', 'record_score', 'typical', 'actual'],
-                size: 1,
-              },
-            },
-          },
-        },
-      },
-    },
-  };
-
-  return client.search(params);
-}
-
-export type AnomaliesResponse = PromiseReturnType<typeof getAnomaliesData>;
+export { ServiceAnomalies };
 export type ConnectionsResponse = PromiseReturnType<typeof getConnectionData>;
 export type ServicesResponse = PromiseReturnType<typeof getServicesData>;
 export type ServiceMapAPIResponse = PromiseReturnType<typeof getServiceMap>;
 
 export async function getServiceMap(options: IEnvOptions) {
-  const [connectionData, servicesData, anomaliesData]: [
+  const [connectionData, servicesData]: [
     // explicit types to avoid TS "excessively deep" error
     ConnectionsResponse,
-    ServicesResponse,
-    AnomaliesResponse
-    // @ts-ignore
-  ] = await Promise.all([
-    getConnectionData(options),
-    getServicesData(options),
-    getAnomaliesData(options),
-  ]);
+    ServicesResponse
```

Comment:
Is the type not inferred?

Reply:
It might be able to be cleaned up now, but at first writing, the compiler would fail with a "excessively deep" TS error.